PureBasicIDE/ColorSchemes.pb

```diff
@@ -0,0 +1,324 @@
+﻿; --------------------------------------------------------------------------------------------
+;  Copyright (c) Fantaisie Software. All rights reserved.
+;  Dual licensed under the GPL and Fantaisie Software licenses.
+;  See LICENSE and LICENSE-FANTAISIE in the project root for license information.
+; --------------------------------------------------------------------------------------------
+
+
+; Store color names in an array, for indexable lookup
+Global Dim ColorName.s(#COLOR_Last_IncludingToolsPanel)
+
+; Color Scheme structure
+Structure ColorSchemeStruct
+  Name.s
```

Comment:
Please use the `Name$` style to declare string variables as it is done in the rest of the IDE sources.